jwql/website/apps/jwql/templates/filesystem_monitor_full.html

```diff
@@ -5,9 +5,9 @@
         <meta charset="utf-8">
         <title>Bokeh Plot</title>
 
-<link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-0.12.5.min.css" type="text/css" />
+<link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
```

Comment:
These versions shouldn't matter (I think) because they'll be updated automatically the next time the filesystem monitor runs.